tests/rptest/tests/raft_availability_test.py

```diff
@@ -95,41 +96,37 @@ def check():
         assert result[0] is not None
         return result[0]
 
-    def _ping_pong(self, timeout=5):
-        kc = KafkaCat(self.redpanda)
-        rpk = RpkTool(self.redpanda)
+    def ping_pong_cold(self):
+        ping_pong = PingPong.cold(self.redpanda.brokers_list(), self.topic, 0,
+                                  self.logger)
+        return ping_pong
 
-        payload = str(random.randint(0, 1000))
-        start = time.time()
-        offset = rpk.produce(self.topic, "tkey", payload, timeout=timeout)
-        consumed = kc.consume_one(self.topic, 0, offset)
-        latency = time.time() - start
-        self.logger.info(
-            f"_ping_pong produced '{payload}' consumed '{consumed}' in {(latency)*1000.0:.2f} ms"
-        )
-        if consumed['payload'] != payload:
-            raise RuntimeError(f"expected '{payload}' got '{consumed}'")
+    def ping_pong_warmed(self):
+        ping_pong = PingPong.warmed(self.redpanda.brokers_list(), self.topic,
+                                    0, self.logger, 20)
+        return ping_pong
 
-    def _is_available(self, timeout=5):
+    def _is_available(self, timeout_s=5):
         try:
             # Should fail
-            self._ping_pong(timeout)
-        except RpkException:
+            self.ping_pong_cold().ping_pong(timeout_s)
+        except:
+            self.logger.exception("expected exception")
```

Comment:
For an expected exception, let's avoid calling `logger.exception` -- it spews backtraces through the log that make it harder for someone to figure out what's going on in a failure.